app/src/ui/branches/branches-container.tsx

```diff
@@ -229,15 +219,23 @@ export class BranchesContainer extends React.Component<
     )
   }
 
-  private onMouseUpNewBranchDrop = () => {
-    if (!this.props.isCherryPickInProgress) {
+  private onMouseUpNewBranchDrop = async () => {
+    const { dragData } = dragAndDropManager
+    if (dragData === null || dragData.type !== DragType.Commit) {
       return
     }
 
-    this.props.dispatcher.setCherryPickCreateBranchFlowStep(
+    await this.props.dispatcher.setCherryPickCreateBranchFlowStep(
       this.props.repository,
       ''
     )
+
+    this.props.dispatcher.showPopup({
```

Comment:
This popup call was in the dispatcher `setCherryPickCreateBranchFlowStep` and was conditional on if it was invoked by a drag event as it can be called for either context menu or drag flow. Since we know this is the drag flow, we can just call it here since it is just another dispatcher call.